api/valor_api/backend/core/geometry.py

```diff
@@ -84,7 +83,6 @@ def get_annotation_type(
             .join(models.Dataset, models.Dataset.id == models.Datum.dataset_id)
             .where(
                 models.Datum.dataset_id == dataset.id,
-                models.Annotation.task_type == task_type.value,
```

Comment:
This change is going to make it always returns raster if a semantic segmentation is stored with a bounding box detection. This will cause issues for the coco use case

Reply:
We have two options here
1. Enforce a singular data type per model
2. Store an inferred data type in the db to stratify over

Reply:
> semantic segmentation is stored with a bounding box detection.

not sure what you mean here as semantic segmentation annotations can't be stored with a bounding box detection; it'll throw a value error in `_match_annotation_to_implied_task_type`

the coco notebook seemed to run fine. can you provide an example where this would fail?

Reply:
The coco notebook currently splits semantic segmenation and object detection into their own models so this doesnt happen.

If you store them in the same model the return of this function is going to always be raster even for the obj det task.

Since the filtering has no way of seeing a difference the evaluation for obj detection will be run over the sem seg rasters.